<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prod</class>
 <widget class="QDialog" name="prod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#prod
{
border-image: url(:/new/prefix1/source/sheeps.jpg);

}</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>821</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>produits physiques</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: #333;
border-radius:15px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>120</y>
        <width>481</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>320</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_2{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_2:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>121</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_3{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_3:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>100</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_4{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_4:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>320</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>83</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 12px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>disponible</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>111</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 12px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>non disponible</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 30px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>gestion des produits physiques</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#333;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#333;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>produits chimiques</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>811</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-color: #333;
border-radius:15px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>quantite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>251</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_5{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_5:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_6{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_6:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_7{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_7:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>360</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_8{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#pushButton_8:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>360</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>421</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 30px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>gestion des produits chimiques</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #333</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#333;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
     </property>
     <property name="placeholderText">
      <string>nom</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
